### PR TITLE
Part1 todo rank ej

### PR DESCRIPTION
**Pull request by xtaje**
For part 1:
EJ1 -> should/must fix
EJ2 -> nice-to-have
EJ3 -> minor

Will follow up with part 2.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -10,7 +10,7 @@ update-code:
 	./checkout-branches-for-ci.py
 
 count-todos:
-	ls *.asciidoc | xargs grep -c TODO | sed  s/:/\\t/
+	ls *.asciidoc | xargs grep -c TODO | sed  s/:/"   "/
```
Comment (hjwp) [inline Makefile]: did some editor plugin do this automatically lol?  i use the tabs to be able to paste terminal output into google sheets...

Comment (hjwp) [inline Makefile]:
```suggestion
	ls *.asciidoc | xargs grep -c TODO | sed  s/:/\\t/
```

Comment (xtaje) [reply Makefile]: Oops.  I did that on purpose and committed it by mistake. Apparently sed handles `\t` on different platforms, so it wasn't working on my Mac.

**Comment by hjwp** [comment]
much appreciated, thanks Ed!

**Pull request merged**